### ISSUE TITLE
A1469 Try Again

### ISSUE THREAD

**Issue by AztekGod** [NSSpiral/Blackb0x#39]
I have the A1469, I right click on the app, open it, it tells me to put it on DFU Mode, I do and after like 10 seconds the screen changes to Try Again on Top with the same message, Hold Menu and Down.  I do it again and it goes back to the same issue.  I had restore it a few times with the same outcome.  Any ideas?  I have a MacBook Pro 2017, and I also tried with a Mac Pro 2010 with the same results.

**Comment by fratinize** [comment]
I have a similar issue, I am using a A1378 v2 App sees it, I click on Jailbreak, it shows my the Apple Remote and Tells me to Hold the Down and Menu Button until Led flashes rapidly, it does for a few seconds then it is Solid again, Just Spinning on App. Another Attempt I received an Extra Button with the Jailbreak, that says Tethered. Yet another Attempt When I press Jailbreak it tells me to Hold the Menu and Play/Pause until bliss. This Does not ever Blink rapidly Just Throwing this out there. Thoughts?

**Comment by NSSpiral** [comment]
> I have a similar issue, I am using a A1378 v2 App sees it, I click on Jailbreak, it shows my the Apple Remote and Tells me to Hold the Down and Menu Button until Led flashes rapidly, it does for a few seconds then it is Solid again, Just Spinning on App. Another Attempt I received an Extra Button with the Jailbreak, that says Tethered. Yet another Attempt When I press Jailbreak it tells me to Hold the Menu and Play/Pause until bliss. This Does not ever Blink rapidly Just Throwing this out there. Thoughts?


Hold Down + Menu (Let go after brief flickers) THEN immediately:
Hold Menu + Play (Until continuous flashes)

2 button combos, it is confusing I know.

Once it is in DFU mode you can just click Jailbreak and then Tethered Boot (assuming you're on 7.1.2, otherwise it is untethered).

**Comment by NSSpiral** [comment]
> I have the A1469, I right click on the app, open it, it tells me to put it on DFU Mode, I do and after like 10 seconds the screen changes to Try Again on Top with the same message, Hold Menu and Down. I do it again and it goes back to the same issue. I had restore it a few times with the same outcome. Any ideas? I have a MacBook Pro 2017, and I also tried with a Mac Pro 2010 with the same results.

This is an unresolved bug, you can get it working by completely powering down the Apple TV and restarting blackb0x. It happens randomly.

**Comment by fratinize** [comment]
I have Been Successful in That I actually Started Kodi. But When moved to its original tv, it never came back, just a Blinking white light on the Apple TV 2 box and a blank screen. Restored with iTunes. On that I am Sure you are aware blackb0x starts iTunes. I have Gone down this Road 5 Times yesterday, Did get it once, but like I said It never came back. All of my Attempts, There was never A Moment the Message said Connect to Your TV. In Fact, it always says Waiting for Apple TV to Restart, but I can Click on the Icon and The Second time have the option to  Jailbreak or Teether boot, After 3 or 4 times Clicking Jailbreak, there does come a Time it says This is Jailbroken, and Gives the Option to Teether Boot, I take that and Have Failed since. I have Tried the Teether boot and You can see it work, But once Again it never says it is completed. Usually about the Third time clicking Jailbreak, the App will crash. Of Course You may be aware that At Some point when The Device is Jailbroken, Blackb0x will just Quit upon Opening. At Which point if I do not see Jodi after 30 Minutes, I will Restore and Head down this Path again. May Times During this Process I am Asked to Press Menu and Down Button and there is not Response, but if I press Menu and Play/pause it Moves on. I have Collected the Logs from this Recent Install that I am Attempting if you would like them.

So After 10 Attempts, Finally Blackb0x Crashes on Startup because it says my Apple TV is Jailbroken, However, 45 Minutes Later No Kodi or Nitro. Since Ihave Installed it Successfully, I will of Course Go down this Road again since I am Stubborn, but if You could Improve the Logs and Steps/Expectations after Each Step would help. I am Not a Noob, but Certainly not Anywhere near a Professional.

Like to Suggest the Blacb0x Window? Allow to Resize. If I use the Terminal, the App Window Stays on Top and I can Not Resize and See Terminal Window fully. thank your your creation and support.

**Comment by fratinize** [comment]
Still At it. Reached the Point where blackb0x Crashes and Says the Apple TV is jailbroken ✅Device connected (AppleTV2,1, Normal)
Jailbroken: Yes
Abort trap: 6
logout
Saving session...
...copying shared history...
...saving history...truncating history files...
...completed.

[Process completed]
but Still do not See kodi
In the Logs I see 
iBootPatcher: Starting...
iBootPatcher: The supplied image is not a valid 32-bit iBoot.
input_path /Users/fmp/Documents/Blackb0x/AppleTV2,1_5.2.1_10B329a_Restore/Firmware/dfu/iBSS.k66ap.RELEASE.patcheddecrypt 0 
error: cannot open infile
img3.c:createAbstractFileFromImg3:463: a13d326d1138702806469990c2fa4d39bf0e82d01a5e7bb6c1321cbb97fbe18ed59229b95dbe96859f1ba121c9fd9036
img3.c:createAbstractFileFromImg3:463: a13d326d1138702806469990c2fa4d39bf0e82d01a5e7bb6c1321cbb97fbe18ed59229b95dbe96859f1ba121c9fd9036
Segmentation fault: 11
logout
Saving session...
...copying shared history...
...saving history...truncating history files...
...completed.

[Process completed]

**Comment by fratinize** [comment]
Sorry Should note (after reading some other issues) that I am running High Sierra

**Comment by zbeebo** [comment]
I spent over 4 hours trying to get into the DFU mode and I finally figured it out.
I was using a non-original remote control, it seems that fake remotes cannot send the reset/DFU commands.
I had to get the original remote from a friend and everything worked like a charm.
Kodi is now installed in less than 5 min.
I hope this helps.
Good luck to everyone

**Comment by fratinize** [comment]
I am using Original Remote and Seem to get into DFU. But I keep Failing. What I am seeing a lot is this Error
iBootPatcher: Starting...
iBootPatcher: The supplied image is not a valid 32-bit iBoot.
input_path /Users/fmp/Documents/Blackb0x/AppleTV2,1_5.2.1_10B329a_Restore/Firmware/dfu/iBSS.k66ap.RELEASE.patcheddecrypt 0 
error: cannot open infile
img3.c:createAbstractFileFromImg3:463: a13d326d1138702806469990c2fa4d39bf0e82d01a5e7bb6c1321cbb97fbe18ed59229b95dbe96859f1ba121c9fd9036
img3.c:createAbstractFileFromImg3:463: a13d326d1138702806469990c2fa4d39bf0e82d01a5e7bb6c1321cbb97fbe18ed59229b95dbe96859f1ba121c9fd9036
Segmentation fault: 11
logout
Saving session...
...copying shared history...
...saving history...truncating history files...
...completed.

[Process completed]

**Comment by NSSpiral** [comment]
> I am using Original Remote and Seem to get into DFU. But I keep Failing. What I am seeing a lot is this Error
> iBootPatcher: Starting...
> iBootPatcher: The supplied image is not a valid 32-bit iBoot.
> input_path /Users/fmp/Documents/Blackb0x/AppleTV2,1_5.2.1_10B329a_Restore/Firmware/dfu/iBSS.k66ap.RELEASE.patcheddecrypt 0
> error: cannot open infile
> img3.c:createAbstractFileFromImg3:463: a13d326d1138702806469990c2fa4d39bf0e82d01a5e7bb6c1321cbb97fbe18ed59229b95dbe96859f1ba121c9fd9036
> img3.c:createAbstractFileFromImg3:463: a13d326d1138702806469990c2fa4d39bf0e82d01a5e7bb6c1321cbb97fbe18ed59229b95dbe96859f1ba121c9fd9036
> Segmentation fault: 11
> logout
> Saving session...
> ...copying shared history...
> ...saving history...truncating history files...
> ...completed.
> 
> [Process completed]

I will test Apple TV 2 stuff on older macOS today. Hopefully I find out why it does this.

**Comment by NSSpiral** [comment]
Not getting this problem. Would you mind pasting the entire log?

**Comment by fratinize** [comment]
The logs I have are from running the terminal version, so I have all five attempts in 5 different files 
is there a place there is a single log file?

**Comment by fratinize** [comment]
[Apple Error Report.txt](https://github.com/NSSpiral/Blackb0x/files/6840562/Apple.Error.Report.txt)
[Apple TV 2 Install 1.txt](https://github.com/NSSpiral/Blackb0x/files/6840563/Apple.TV.2.Install.1.txt)
[Apple TV 2 Install 2.txt](https://github.com/NSSpiral/Blackb0x/files/6840564/Apple.TV.2.Install.2.txt)
[Apple TV 2 Install 3.txt](https://github.com/NSSpiral/Blackb0x/files/6840565/Apple.TV.2.Install.3.txt)
[Apple TV 2 Install 4.txt](https://github.com/NSSpiral/Blackb0x/files/6840566/Apple.TV.2.Install.4.txt)
[Apple TV 2 Install 5.txt](https://github.com/NSSpiral/Blackb0x/files/6840567/Apple.TV.2.Install.5.txt)
[Apple TV 2 Install 6.txt](https://github.com/NSSpiral/Blackb0x/files/6840568/Apple.TV.2.Install.6.txt)
I have also included one of the Apple Reports that were generated from one of the Fails, not sure if that helps any.
To recap my Process. I connected the Apple TV, Down and Menu, worked fine, Immediately went to menu and Play/pause went just fine. at the end of that run, it said waiting to reboot, but I could still select the Apple TV and Was presented with 2 Buttons jailbreak and teetheredboot, Naturally went with Jailbreak since it did not seem to be jailbroken. Thank you for your help.

**Comment by NSSpiral** [comment]
For Apple TV 2 you need to tethered boot for Kodi/nitoTV to install. They'll also not be visible if it's not tethered booted so it can't be powered off when you're connecting it to the TV. This is probably why it doesn't appear jailbroken. If it's more convenient for you to have an untethered jailbreak I'd recommend installing firmware 5.3 and installing the jailbreak on that. I think the blackb0x interface might be a bit confusing for installing the tethered jailbreak on Apple TV 2 atm.

**Comment by fratinize** [comment]
Hmm
At no time did I power down, remove the USB and left the hdmi in all the time Ethernet cable was never unplugged

Like I mentioned before I did have a successful install seen nitro and kodi. Even opened the app
Then when I moved to it’s original tv it never came back up
I will try it again soon and let ya know ty
So I’m clear click jailbreak the first time only? Or if it is successful I’ll only see tethered option?

**Comment by NSSpiral** [comment]
You connected it to a different TV without powering it down and Kodi/nitoTV
were no longer there?

**Comment by fratinize** [comment]
It never came up, black screen blinking light, for over an hour, got impatient, figured I could do it again.

**Comment by db-rtaylor** [comment]
"It never came up, black screen blinking light, for over an hour, got impatient, figured I could do it again."

This is the point I am at with my A1469. Left it on overnight - black screen slow blinking light. 
Ran BlackB0x again - same thing.

**Comment by NSSpiral** [comment]
I'll try clean up the process for Apple TV 2, hopefully within the next couple of days.

**Comment by YanAndrew** [comment]
Excuse me Guys, both of my attempts on my A1469 (running software 7.8) failed. Now, I cannot even open Blackb0xas it keeps crashing when I try. No Kodi and no NitoTV appear. And now when I connect the A1469 into my computer, iTunes can’t even read the software version. The spinning wheel just keeps going & going.

Any advice / guidance would be hugely appreciated.
